package.json

```diff
@@ -417,6 +417,31 @@
                 "type": "string",
                 "description": "Output path for the binary of delve",
                 "default": "debug"
+              },
```

Comment:
Having a single setting called say `dlvConfig` would be preferable than 5 settings. 
```
 "dlvConfig": {
                "type": "object",
                "properties": {
                  "followPointers": {
                    "type": "boolean",
                    "default": true,
                    "description": "FollowPointers requests pointers to be automatically dereferenced."
                  },
                  "maxVariableRecurse": {
                    "type": "number",
                    "default": 1,
                    "description": "MaxVariableRecurse is how far to recurse when evaluating nested types"
                  },
                  "maxStringLen": {
                    "type": "number",
                    "description": "MaxStringLen is the maximum number of bytes read from a string",
                    "default": 64
                  },
                  "maxArrayValues": {
                    "type": "number",
                    "description": "MaxArrayValues is the maximum number of elements read from an array, a slice or a map",
                    "default": 64
                  },
                  "maxStructFields": {
                    "type": "number",
                    "description": "MaxStructFields is the maximum number of fields read from a struct, -1 will read all fields",
                    "default": -1
                  }
                },
                "default": {
                  "followPointers": true,
                  "maxVariableRecurse": 1,
                  "maxStringLen": 64,
                  "maxArrayValues": 64,
                  "maxStructFields": -1
                },
                "description": "....."
              }
```